support/cas-server-support-pm-ldap/src/main/java/org/apereo/cas/pm/LdapPasswordManagementService.java

```diff
@@ -39,6 +42,16 @@ public LdapPasswordManagementService(final CipherExecutor<Serializable, String>
                                          final PasswordHistoryService passwordHistoryService) {
         super(passwordManagementProperties, cipherExecutor, issuer, passwordHistoryService);
         this.ldapProperties = passwordManagementProperties.getLdap();
+        this.ldapProperties.forEach(ldap -> {
+            this.connectionFactoryMap.put(ldap, LdapUtils.newLdaptiveConnectionFactory(ldap));
+        });
```

Comment:
Pass the connection factory map as a ctor argument; build the map inside the bean

Reply:
See above. I believe make `connectionFactoryMap` into a separate bean is a overkill.